src/StatusQ/Components/StatusListItem.qml

```diff
@@ -187,6 +197,15 @@ Rectangle {
                           return Theme.palette.dangerColor1
                   }
                 }
+
+                MouseArea {
+                    anchors.fill: parent
+                    cursorShape: containsMouse? Qt.PointingHandCursor : Qt.ArrowCursor
+                    hoverEnabled: true
+                    onClicked: {
+                        statusListItem.titleClicked(statusListItem.titleId)
+                    }
+                }
```

Comment:
I see why this is useful, just need to clarify `titleId`, maybe there's a cleaner way to get the same results.